plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -305,7 +305,16 @@ public List<JdbcColumnHandle> getColumns(ConnectorSession session, JdbcTableHand
                         getInteger(resultSet, "DECIMAL_DIGITS"),
                         Optional.empty(),
                         Optional.empty());
-                Optional<ColumnMapping> columnMapping = toColumnMapping(session, connection, typeHandle);
+                Optional<ColumnMapping> columnMapping;
+                try {
+                    columnMapping = toColumnMapping(session, connection, typeHandle);
```

Comment:
I think if jdbc driver is broken then `toColumnMapping` should deal with it per connector basis.